spec.bs

```diff
@@ -1563,3 +1598,133 @@ specification adds another value for fenced frames:
 
 : "`fencedframe`"
 :: This [=navigable=] is displaying a <{fencedframe}>'s content
+
+<h3 id=permissions-policy-changes>Permissions Policies</h3>
+Permissions are granted to fenced frames through the FencedFrameConfig object, which defines the
+required permissions for the fenced frame to load. The fenced frame is only allowed to navigate if
+all of the required permissions are present in the context loading the fenced frame. When a fenced
+frame loads, it is assumed that the permissions checks passed, and the fenced frame will load with
+the list of permissions that were specified in the config object.
+
+Modify the [=Create a Permissions Policy for a navigable=] algorithm:
```

Comment:
What's up with this?

Reply:
I think I just forgot to remove that line.